### ISSUE TITLE
remove degree for participle?

### ISSUE THREAD

**Issue by balmas** [latin-language-toolkit/arethusa-configs#32]
Request from a student of @gcelano : 

Is there any possibility to remove option "degree" when we have to introduce participle? I can't think of any case where this option should be useful for participle.

@gcelano I'm assigning this to you to discuss with the other editors (including @vgorman1 ) and make a decision.

**Comment by vgorman1** [comment]
We should talk about this more before we talk any action.  Bob needs to weigh in.  This is similar to the issue with participles as adverbs.  I give the verb as the lemma, but POS adverb.  We have the issue of whether we are prioritizing derivation or morphology.

Vanessa B. Gorman
Associate Professor
Undergraduate Chair
Department of History
University of Nebraska-Lincoln
619 Oldfather Hall
Lincoln, NE 68588-0327

**Comment by balmas** [comment]
Thanks @vgorman1 - I will wait for consensus before making changes...

**Comment by nevenjovanovic** [comment]
I agree with @gcelano that Latin participle does not need a "degree" category, either morphologically or lexically (if it can be compared, it has become an adjective). Moreover, I would suggest to offer only present and perfect as tenses for Latin participle.
In general, I believe that for the present _morphological_ approach of Arethusa it would be paedagogically sound to mirror the traditional classroom approach as far as possible (insofar as there is one across different national traditions). Also, if we follow the approach of standard lexica in defining lexical units (for example, adverbs), we won't confuse the students which rely on such lexica in their daily work.

**Comment by gcelano** [comment]
I would also keep the future for the participle. This issue reveals - again - that, for a specific kind of annotation, the tagset may offer more options than it should, the main concern having been that it included all the necessary options. Since this change, as well as the others, are going to be "major" changes in the underlying json file, I recommend that they be first made in a separate test file.

**Comment by nevenjovanovic** [comment]
Dear @gcelano , you are definitively right about the future participle (my grammar is a bit influenced by the seaside). Regarding the size of the tagset options, you are right to a degree; we have also to have in mind, however, that a larger number of options increases chance of errors too. Finally, I think that Github issues are just the place where both "major" and "minor" changes can and should be discussed and "tested" (like you did with my rash suggestion) before they are implemented. Best, Neven.

**Comment by balmas** [comment]
@TDBuck can you follow up on this?

**Comment by TDBuck** [comment]
So currently we can remove the degree option from the participle. But creating a narrow list of tenses for the participle would require making a separate list. So if you were to query the treebanks for everything in the perfect tense, you would either get all the perfect participles or all other perfect verbs. 
What we could implement is a "training wheels" tagset which would be designed explicitly for people learning latin. We already have a whole series of tagsets like that with the elmlat series (which I think are tied to chapters in a book) But in terms of the master tagset, I think having a single set of values for tenses is the best solution.
